modules/sbo-methods-of-exposing-binding-data.adoc

```diff
@@ -156,7 +156,21 @@ metadata:
     ...
 ----
 
-.Example: Exposing binding data from a `ConfigMap` object in the `metadata.annotations.dbconfig` custom field
+The previous example places the name of the secret name in the `{.metadata.name}-pguser-{.metadata.name}` template that resolves to `hippo-pguser-hippo` eventually. The template can contain multiple JSONPath.
```

Comment:
```suggestion
The previous example places the name of the secret name in the `{.metadata.name}-pguser-{.metadata.name}` template that resolves to `hippo-pguser-hippo`. The template can contain multiple JSONPath.
```
Maybe cut this word? I don't think the sentence loses any significant meaning by omitting it. It's your call.

Reply:
> Does `multiple JSONPath` need a noun added? `multiple JSONPath values` or `multiple JSONPath fields`?

@pmacik Can you PTAL this comment and let me know your opinion? I am working on the comments so your suggestion will help. Thanks!

Reply:
IMHO, it should be `multiple JSONPath expressions`.

Reply:
See https://docs.oracle.com/cd/E60058_01/PDF/8.0.8.x/8.0.8.0.0/PMF_HTML/JsonPath_Expressions.htm for example reference

Reply:
Thanks @pmacik, this is helpful :+1: